documentation/src/main/asciidoc/topics/ref_redis_commands.adoc

```diff
@@ -159,6 +159,16 @@ NOTE: This implementation attempts to return all attributes that a real Redis se
 * link:https://redis.io/commands/json.forget[JSON.FORGET]
 
 * link:https://redis.io/commands/json.get[JSON.GET]
++
+[NOTE]
+=====
+ JSON filter with RegEx differencies from Redis implementation:
+
+1. Pattern could be expressed in Redis style "(i)pattern" or Java style "/.*pattern.*/i)".
+2. Dynamic filter with regex is not supported. i.e. `$.store.book[?(@.name < $.pattern)]` doesn't work.
```

Comment:
Do you know why it doesn't work? Is there a missing supported in the used library?

Reply:
I've created an issue: https://github.com/json-path/JsonPath/issues/1062

Reply:
Looking at the README on JsonPath there's an example that seems to suggest it should work: `$..book[?(@.price <= $['expensive'])]`: https://github.com/json-path/JsonPath/blob/master/README.md?plain=1#L161